### DISCUSSION TITLE
Unifi LXC isn't updating the Unifi app

### DISCUSSION THREAD

**Discussion by sendmebits** [tteck/Proxmox#1944] [Help Needed] [answered]
The Unifi software doesn't seem to be updating past 7.3.83, there's a bunch of versions past this though as per [here](https://community.ui.com/questions/UniFi-Installation-Scripts-or-UniFi-Easy-Update-Script-or-UniFi-Lets-Encrypt-or-UniFi-Easy-Encrypt-/ccbc7530-dd61-40a7-82ec-22b17f027776) with the newest being [7.5.187](https://community.ui.com/releases/UniFi-Network-Application-7-5-187/408b64c5-a485-4a37-843c-31e87140be64)

Here is the output of the update script:
```
Get:1 http://security.debian.org bullseye-security InRelease [48.4 kB]
Hit:2 http://deb.debian.org/debian bullseye InRelease                                                                                                   
Get:3 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]                                                             
Ign:5 https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 InRelease                                                 
Hit:6 https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 Release
Hit:4 https://dl.ui.com/unifi/debian unifi-7.3 InRelease
Fetched 92.4 kB in 1s (88.7 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
unifi is already the newest version (7.3.83-19645-1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
 ✓ Updated Successfully
```

Let me know if I've missed something or if I can provide more info to help root cause.

**Comment by tteck** [comment, marked as answer]
It appears that your current setup is based on an older installation that doesn't utilize the updated Unifi repository. I recommend creating a backup and then creating a new Unifi LXC container, followed by restoring your backup to this fresh environment.

```
root@unifi:~# apt update
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://security.debian.org bookworm-security InRelease                           
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease                          
Hit:4 https://dl.ui.com/unifi/debian stable InRelease
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
```
![Screenshot 2023-10-20 5 17 49 AM](https://github.com/tteck/Proxmox/assets/21342670/27921d81-f2dc-4e84-aff3-30d63efd1c7f)

**Reply by sendmebits** [reply]
Done - I've backed up Unifi, redeployed the newest LXC and restored my config. All seems to be working now, thanks!

**Comment by Scrat80** [comment]
How'd you get the Bookworm release? Just the usual apt update, or did you start off with Bookworm from scratch?
